data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
-set -eo pipefail
```

Comment:
This completely removes error checking, no?  What happens if, for example `mkdir -p "${ARTIFACTS}/bootstrap/journals"` fails and we continue to launch all the commands that attempt to fill it in?

Reply:
I think we should try to make sure this script never fails and gathers as much as it can.

Reply:
also the bootstrap node is very controlled env so things like mkdir failing, we'll probably catch in our CI ;)

Reply:
> also the bootstrap node is very controlled env so things like mkdir failing, we'll probably catch in our CI ;)

*If* we look and notice that things are missing.  I'm unlikely to be that observant unless quite a lot is missing ;).